### PR TITLE
📝 Document inactive environments

### LINKED ISSUE #2092: :memo: Document inactive environments
## What needs to be documented?
 
We don't have clear docs for what inactive environments are and how to work with them. For example, if there are branches on GitHub that aren't running (such as never had a PR opened and the settings are for only PR environments to be active), users have not seen the environments in the dropdown to select environments and been confused by trying to push a branch with the same name and having it be rejected.

Make it clear what inactive environments are, how they might come to be without user action, and how to work with them. Include:

* ~Where to find them in the console~
* How to see the status of an environment
* ~How to activate in CLI/console~
* See if there's other places to link the info (like source integration)

[Additional context](https://github.com/orgs/platformsh/projects/3)

### PR DESCRIPTION
**Pull request by CollierCZ**
## Why

Closes #2092

## What's changed

Added how to get the status and a source code for describing on all of the source integrations. (The entire page for each source integration likely should be a shortcode, but it was taking longer to verify steps and seemed out of scope for this task.

### CODE REVIEW

File: docs/layouts/shortcodes/integration-environment-status.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{{ $source := .Get "source" }}
+{{ $pull := "pull"}}
+{{ if eq $source "GitLab" }}
+  {{ $pull = "merge" }}
+{{ end }}
+## Environment parent and status
+
+Environments based on {{ $source }} **{{ $pull }} requests** have their target branch set as their parent on Platform.sh.
+They're added as [activate environments](/glossary.html#active-environment) with a copy of the parent's data.
```
Comment (trolologuy) [inline docs/layouts/shortcodes/integration-environment-status.md:9]: :clipboard:
```suggestion
They're added as [active environments](/glossary.html#active-environment) with a copy of the parent's data.
```

File: docs/layouts/shortcodes/integration-environment-status.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+Environments based on **branches** always have the default branch set as their parent on Platform.sh.
+They're added as [inactivate environments](/glossary.html#inactive-environment) with no data or services.
```
Comment (trolologuy) [inline docs/layouts/shortcodes/integration-environment-status.md:12]: :clipboard:
```suggestion
They're added as [inactive environments](/glossary.html#inactive-environment) with no data or services.
```

File: docs/src/environments/_index.md [hunk 3]
```diff
@@ -138,3 +138,22 @@ Staging
...
+
+* [Active](../other/glossary.md#active-environment):
+  A deployed environment with services and data.
+* [Inactive](../other/glossary.md#inactive-environment):
+  An environment that isn't deployed and has no services or data, only code.
+
+You can see an environment's status in multiple ways:
+
+* In the console:
+  * Inactive environments are lighter in the environment list and don't appear in the environment selector.
```
Comment (trolologuy) [inline docs/src/environments/_index.md:154]: :nail_care:
We could add the link to [the environment configuration](https://pr-2328-nkae3aq-652soceglkw4u.eu-3.platformsh.site/administration/web/configure-environment.html).

**Review by trolologuy — CHANGES_REQUESTED** [review]
Couple of typos to fix before merging.

- [x] Final built pages are working as expected.
- [x] The information is correct.
- [x] The information is comprehensive.
- [x] The information is concise.
- [x] The information is clear.

File: docs/src/environments/_index.md (hunk 3 above)
Comment (CollierCZ) [reply docs/src/environments/_index.md:154]: I added it to the top-level list item, if you think that works.

Comment (trolologuy) [reply docs/src/environments/_index.md:154]: Yes, that works perfectly, thanks 🙏🏼

**Review by trolologuy — APPROVED** [review]
✅  🚀

**Pull request merged**